cmd/meroxa/root/auth/whoami.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright © 2021 Meroxa Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or impliew.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package auth
+
+import (
+	"context"
+
+	"github.com/meroxa/cli/cmd/meroxa/builder"
+
+	"github.com/meroxa/cli/log"
+
+	"github.com/meroxa/meroxa-go"
```

Comment:
I heard you like empty lines.. :trollface:

Reply:
@lovromazgon blame my IDE! Those are automatically added and since imports are usually collapsed I didn't even notice 😀